src/file.ts

```diff
@@ -35,13 +35,15 @@ async function zipBuildResult(buildResult: BuildResult): Promise<void> {
 }
 
 async function moveBuildsToExportDirectory(buildResults: BuildResult[], moveArchived?: boolean): Promise<void> {
-  const fullExportPath = path.resolve(RELATIVE_EXPORT_PATH);
-  core.startGroup(`Moving exports to ${fullExportPath}`);
-
-  await io.mkdirP(fullExportPath);
-
   const promises: Promise<void>[] = [];
   for (const buildResult of buildResults) {
+    const fullExportPath = path.resolve(
+      USE_PRESET_EXPORT_PATH ? path.dirname(buildResult.preset.export_path) : RELATIVE_EXPORT_PATH,
```

Comment:
@firebelley: "This could be an issue for users with lots of presets (I have 8 configured presets on one of my projects!)"
What would be the problem here? I moved the path inside the loop so each export gets moved to where it is defined. This is even an improvement because before you couldn't move them to different paths.

Reply:
I'm saying it would a problem in general if I had a large number of export presets and I wanted them to all go to the same place, I'd have to manually change the paths for each one of them. That's a non-issue if we're keeping `relative_export_path`

Reply:
Ah, okay. But I think that it's not really a problem at all. The action already requires you to set a path in the export_presets. So it shouldn't be much additional work if we think from a new users perspective. I do see the minor problem of migrating to this. But overall I don't think that this is big enough of a problem as it's not intense manual work but rather adding that folder to the path in export_presets.